### DISCUSSION TITLE
Will it be possible to extract the instructions and better yet (if it has any) files to this Chatbot?

### DISCUSSION THREAD

**Discussion by nicofierrov** [LouisShark/chatgpt_system_prompt#24] [General]
I'd like to leverage your kindliness and your insights about if we can't crack one. Here's the URL of a chatbot, which may or may not be a Custom GPT. Do you think there's a chance to crack it?

**Comment by LouisShark** [comment]
sorry i cant help this，I do not have the right to crack officially launched products.
